js/util/struct_array.js

```diff
@@ -204,15 +261,20 @@ const structArrayTypeCache = {};
  *
  * @private
  */
-function createStructArrayType(options) {
+
+function createStructArrayType(options: {|
+  members: Array<{type: ViewType, name: string, components?:? number}>,
+  alignment?: number
+|}) {
 
     const key = JSON.stringify(options);
 
     if (structArrayTypeCache[key]) {
         return structArrayTypeCache[key];
     }
 
-    if (options.alignment === undefined) options.alignment = 1;
+    const alignment = (options.alignment === undefined || options.alignment === null) ?
```

Comment:
It would be equivalent and more concise to write `options.alignment == null` instead of `(options.alignment === undefined || options.alignment === null)`. We have explicitly allowed this one use of the `==` operator in the GL JS codebase.